### PR TITLE
fix: dialcode is changed to country code in entry form

### PR DESCRIPTION
**Pull request by Tapudp**
## Motivation
https://github.com/reach4help/reach4help/issues/831

## Release Checklist

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

### CODE REVIEW

File: map/src/components/add-information.tsx [hunk 1]
```diff
@@ -714,10 +714,10 @@ class AddInstructions extends React.Component<Props, State> {
             // Remove array completely if empty
             if (arr.length === 0) {
               // eslint-disable-next-line no-param-reassign
-              delete (info.contact[type] as ContactDetails)[method];
```
Comment (mcverter) [inline map/src/components/add-information.tsx:717]: where did this come from?

Comment (Tapudp) [reply map/src/components/add-information.tsx:717]: its more like a lint:fix or prettier thing than anything else

Comment (mcverter) [reply map/src/components/add-information.tsx:717]: not sure what this is from.  this file should not have been changed by your push.  please start again by forking the repo first.   set your upstream to reach4help/reach4help.  always do git merge upstream/development  before you push.

**Comment by mcverter** [comment]
@Tapudp I am closing this pull request.  You should not have changed both files.  Please start again by creating a new fork.

**Pull request closed without merging**